### DISCUSSION TITLE
Clusters and NavigationItems

### DISCUSSION THREAD

**Discussion by florentin1701** [filamentphp/filament#13176] [Help]
### Package

Panel builder

### Package Version

v3.2.83

### How can we help you?

I'm currently using a page which contains sub navigations items in my resource like so :

```
public static function getRecordSubNavigation(Page $page): array
    {
        return $page->generateNavigationItems([
            Pages\Page1::class,
            Pages\Page2::class,
            Pages\Page3::class,
            Pages\Page4::class,
            Pages\Page5::class,
        ]);
    }
```
        
First of all, I'd like to know if it's possible to make the sidebar collapsible ? 
    
And secondly, when I specify a navigationGroup for X pages and a navigationSort for each of them, the ones not in a group always come first and the group comes last despite that I put lower order rank.

Any ideas if it's possible in the current version ?

**Comment by dave-mills** [comment]
+1 for the question about putting navigation items outside of a group after grouped items. I don't think it's possible, but if it is, I'd be keen to know how.
